from PyQt5 import QtCore, QtGui, QtWidgets

class Ui_profile(object):
    def setupUi(self, profile):
        profile.setObjectName("profile")
        profile.resize(1134, 867)
        self.centralwidget = QtWidgets.QWidget(profile)
        self.centralwidget.setObjectName("centralwidget")
        self.frame = QtWidgets.QFrame(self.centralwidget)
        self.frame.setGeometry(QtCore.QRect(-1, -1, 1131, 841))
        self.frame.setStyleSheet("\n"
"background-color: rgb(225, 0, 0);\n"
"border-radius:15px;")
        self.frame.setFrameShape(QtWidgets.QFrame.StyledPanel)
        self.frame.setFrameShadow(QtWidgets.QFrame.Raised)
        self.frame.setObjectName("frame")
        self.frame_2 = QtWidgets.QFrame(self.frame)
        self.frame_2.setGeometry(QtCore.QRect(210, 100, 431, 671))
        self.frame_2.setStyleSheet("background-color: rgb(255, 255, 255);\n"
"border-radius:15px;")
        self.frame_2.setFrameShape(QtWidgets.QFrame.StyledPanel)
        self.frame_2.setFrameShadow(QtWidgets.QFrame.Raised)
        self.frame_2.setObjectName("frame_2")
        self.lineEdit = QtWidgets.QLineEdit(self.frame_2)
        self.lineEdit.setGeometry(QtCore.QRect(20, 240, 391, 51))
        font = QtGui.QFont()
        font.setFamily("Segoe UI Semilight")
        font.setPointSize(12)
        self.lineEdit.setFont(font)
        self.lineEdit.setStyleSheet("background-color: #d8d8d8;")
        self.lineEdit.setObjectName("lineEdit")
        self.lineEdit_2 = QtWidgets.QLineEdit(self.frame_2)
        self.lineEdit_2.setGeometry(QtCore.QRect(20, 310, 391, 51))
        font = QtGui.QFont()
        font.setFamily("Segoe UI Semilight")
        font.setPointSize(12)
        self.lineEdit_2.setFont(font)
        self.lineEdit_2.setStyleSheet("background-color:#d8d8d8;")
        self.lineEdit_2.setObjectName("lineEdit_2")
        self.lineEdit_3 = QtWidgets.QLineEdit(self.frame_2)
        self.lineEdit_3.setGeometry(QtCore.QRect(20, 380, 391, 51))
        font = QtGui.QFont()
        font.setFamily("Segoe UI Semilight")
        font.setPointSize(12)
        self.lineEdit_3.setFont(font)
        self.lineEdit_3.setStyleSheet("background-color: #d8d8d8;")
        self.lineEdit_3.setObjectName("lineEdit_3")
        self.pushButton = QtWidgets.QPushButton(self.frame_2)
        self.pushButton.setGeometry(QtCore.QRect(130, 480, 181, 61))
        font = QtGui.QFont()
        font.setFamily("Segoe UI Semibold")
        font.setPointSize(20)
        self.pushButton.setFont(font)
        self.pushButton.setCursor(QtGui.QCursor(QtCore.Qt.PointingHandCursor))
        self.pushButton.setStyleSheet("background-color: rgb(255, 0, 0);")
        self.pushButton.setObjectName("pushButton")
        self.image = QtWidgets.QPushButton(self.frame_2)
        self.image.setGeometry(QtCore.QRect(100, 30, 201, 161))
        self.image.setCursor(QtGui.QCursor(QtCore.Qt.CrossCursor))
        self.image.setText("")
        icon = QtGui.QIcon()
        icon.addPixmap(QtGui.QPixmap("../Pictures/icons8-picture-100.png"), QtGui.QIcon.Normal, QtGui.QIcon.Off)
        self.image.setIcon(icon)
        self.image.setIconSize(QtCore.QSize(200, 200))
        self.image.setDefault(False)
        self.image.setObjectName("image")
        self.frame_4 = QtWidgets.QFrame(self.frame)
        self.frame_4.setGeometry(QtCore.QRect(0, 0, 1131, 91))
        self.frame_4.setStyleSheet("background-color: rgb(255, 0, 0);")
        self.frame_4.setFrameShape(QtWidgets.QFrame.StyledPanel)
        self.frame_4.setFrameShadow(QtWidgets.QFrame.Raised)
        self.frame_4.setObjectName("frame_4")
        self.close = QtWidgets.QPushButton(self.frame_4)
        self.close.setGeometry(QtCore.QRect(1090, 10, 31, 28))
        font = QtGui.QFont()
        font.setFamily("MS Gothic")
        self.close.setFont(font)
        self.close.setCursor(QtGui.QCursor(QtCore.Qt.PointingHandCursor))
        self.close.setStyleSheet("background-color: rgb(193, 0, 0);\n"
"border-radius:8px;\n"
"")
        self.close.setText("")
        self.close.setObjectName("close")
        self.label_11 = QtWidgets.QLabel(self.frame_4)
        self.label_11.setGeometry(QtCore.QRect(520, 0, 111, 101))
        self.label_11.setText("")
        self.label_11.setPixmap(QtGui.QPixmap("../Pictures/icons8-youtube-512.png"))
        self.label_11.setScaledContents(True)
        self.label_11.setObjectName("label_11")
        self.label = QtWidgets.QLabel(self.frame_4)
        self.label.setGeometry(QtCore.QRect(70, 30, 111, 41))
        font = QtGui.QFont()
        font.setFamily("Raleway")
        font.setPointSize(17)
        self.label.setFont(font)
        self.label.setObjectName("label")
        self.label_9 = QtWidgets.QLabel(self.frame_4)
        self.label_9.setGeometry(QtCore.QRect(10, 20, 55, 61))
        self.label_9.setText("")
        self.label_9.setPixmap(QtGui.QPixmap("../Pictures/icons8-customer-90.png"))
        self.label_9.setScaledContents(True)
        self.label_9.setObjectName("label_9")
        self.max = QtWidgets.QPushButton(self.frame_4)
        self.max.setGeometry(QtCore.QRect(1050, 10, 31, 28))
        self.max.setCursor(QtGui.QCursor(QtCore.Qt.PointingHandCursor))
        self.max.setStyleSheet("background-color: rgb(255, 255, 0);\n"
"border-radius:8px;")
        self.max.setText("")
        icon1 = QtGui.QIcon()
        icon1.addPixmap(QtGui.QPixmap("../YOUTUBE_DOWNLOADER/Pictures/icons8-subtract-90.png"), QtGui.QIcon.Normal, QtGui.QIcon.Off)
        self.max.setIcon(icon1)
        self.max.setObjectName("max")
        self.min = QtWidgets.QPushButton(self.frame_4)
        self.min.setGeometry(QtCore.QRect(1010, 10, 31, 28))
        self.min.setCursor(QtGui.QCursor(QtCore.Qt.PointingHandCursor))
        self.min.setStyleSheet("background-color: rgb(0, 229, 0);\n"
"border-radius:8px;")
        self.min.setText("")
        self.min.setIcon(icon1)
        self.min.setObjectName("min")
        self.frame_5 = QtWidgets.QFrame(self.frame)
        self.frame_5.setGeometry(QtCore.QRect(0, 90, 191, 751))
        self.frame_5.setStyleSheet("background-color: rgb(255, 0, 0);")
        self.frame_5.setFrameShape(QtWidgets.QFrame.StyledPanel)
        self.frame_5.setFrameShadow(QtWidgets.QFrame.Raised)
        self.frame_5.setObjectName("frame_5")
        self.pushButton_2 = QtWidgets.QPushButton(self.frame_5)
        self.pushButton_2.setGeometry(QtCore.QRect(0, 0, 191, 61))
        font = QtGui.QFont()
        font.setFamily("Raleway SemiBold")
        font.setPointSize(14)
        self.pushButton_2.setFont(font)
        self.pushButton_2.setCursor(QtGui.QCursor(QtCore.Qt.PointingHandCursor))
        self.pushButton_2.setToolTip("")
        self.pushButton_2.setStyleSheet("background-color: rgb(0, 0, 0);\n"
"color: rgb(255, 0, 0);\n"
"border-radius:0px;")
        self.pushButton_2.setIconSize(QtCore.QSize(50, 50))
        self.pushButton_2.setObjectName("pushButton_2")
        self.pushButton_3 = QtWidgets.QPushButton(self.frame_5)
        self.pushButton_3.setGeometry(QtCore.QRect(0, 80, 191, 61))
        font = QtGui.QFont()
        font.setFamily("Raleway SemiBold")
        font.setPointSize(14)
        self.pushButton_3.setFont(font)
        self.pushButton_3.setCursor(QtGui.QCursor(QtCore.Qt.PointingHandCursor))
        self.pushButton_3.setToolTip("")
        self.pushButton_3.setStyleSheet("background-color: rgb(0, 0, 0);\n"
"color: rgb(255, 0, 0);\n"
"border-radius:0px;")
        self.pushButton_3.setIconSize(QtCore.QSize(60, 60))
        self.pushButton_3.setObjectName("pushButton_3")
        self.pushButton_4 = QtWidgets.QPushButton(self.frame_5)
        self.pushButton_4.setGeometry(QtCore.QRect(0, 160, 191, 61))
        font = QtGui.QFont()
        font.setFamily("Raleway SemiBold")
        font.setPointSize(14)
        self.pushButton_4.setFont(font)
        self.pushButton_4.setCursor(QtGui.QCursor(QtCore.Qt.PointingHandCursor))
        self.pushButton_4.setToolTip("")
        self.pushButton_4.setStyleSheet("background-color: rgb(0, 0, 0);\n"
"color: rgb(255, 0, 0);\n"
"border-radius:0px;")
        self.pushButton_4.setIconSize(QtCore.QSize(60, 60))
        self.pushButton_4.setObjectName("pushButton_4")
        self.label_10 = QtWidgets.QLabel(self.frame_5)
        self.label_10.setGeometry(QtCore.QRect(0, 720, 191, 21))
        self.label_10.setStyleSheet("background-color: rgb(0, 0, 0);")
        self.label_10.setText("")
        self.label_10.setScaledContents(True)
        self.label_10.setObjectName("label_10")
        self.label_12 = QtWidgets.QLabel(self.frame_5)
        self.label_12.setGeometry(QtCore.QRect(0, 690, 191, 21))
        self.label_12.setStyleSheet("background-color: rgb(0, 0, 0);")
        self.label_12.setText("")
        self.label_12.setScaledContents(True)
        self.label_12.setObjectName("label_12")
        self.label_13 = QtWidgets.QLabel(self.frame_5)
        self.label_13.setGeometry(QtCore.QRect(0, 660, 191, 21))
        self.label_13.setStyleSheet("background-color: rgb(0, 0, 0);")
        self.label_13.setText("")
        self.label_13.setScaledContents(True)
        self.label_13.setObjectName("label_13")
        self.label_14 = QtWidgets.QLabel(self.frame_5)
        self.label_14.setGeometry(QtCore.QRect(0, 630, 191, 21))
        self.label_14.setStyleSheet("background-color: rgb(0, 0, 0);")
        self.label_14.setText("")
        self.label_14.setScaledContents(True)
        self.label_14.setObjectName("label_14")
        self.label_15 = QtWidgets.QLabel(self.frame_5)
        self.label_15.setGeometry(QtCore.QRect(0, 600, 191, 21))
        self.label_15.setStyleSheet("background-color: rgb(0, 0, 0);")
        self.label_15.setText("")
        self.label_15.setScaledContents(True)
        self.label_15.setObjectName("label_15")
        self.label_2 = QtWidgets.QLabel(self.frame)
        self.label_2.setGeometry(QtCore.QRect(660, 320, 451, 451))
        self.label_2.setText("")
        self.label_2.setPixmap(QtGui.QPixmap("../Pictures/Phone maintenance-bro.svg"))
        self.label_2.setScaledContents(True)
        self.label_2.setObjectName("label_2")
        self.frame_3 = QtWidgets.QFrame(self.frame)
        self.frame_3.setGeometry(QtCore.QRect(580, 100, 541, 211))
        self.frame_3.setStyleSheet("background-color: rgb(255, 255, 255);\n"
"border-radius:15px;")
        self.frame_3.setFrameShape(QtWidgets.QFrame.StyledPanel)
        self.frame_3.setFrameShadow(QtWidgets.QFrame.Raised)
        self.frame_3.setObjectName("frame_3")
        self.label_3 = QtWidgets.QLabel(self.frame_3)
        self.label_3.setGeometry(QtCore.QRect(10, 30, 191, 31))
        font = QtGui.QFont()
        font.setFamily("Segoe UI Semibold")
        font.setPointSize(14)
        self.label_3.setFont(font)
        self.label_3.setObjectName("label_3")
        self.label_4 = QtWidgets.QLabel(self.frame_3)
        self.label_4.setGeometry(QtCore.QRect(10, 90, 151, 21))
        font = QtGui.QFont()
        font.setFamily("Segoe UI Semibold")
        font.setPointSize(14)
        self.label_4.setFont(font)
        self.label_4.setObjectName("label_4")
        self.label_5 = QtWidgets.QLabel(self.frame_3)
        self.label_5.setGeometry(QtCore.QRect(10, 150, 191, 21))
        font = QtGui.QFont()
        font.setFamily("Segoe UI Semibold")
        font.setPointSize(14)
        self.label_5.setFont(font)
        self.label_5.setObjectName("label_5")
        self.label_6 = QtWidgets.QLabel(self.frame_3)
        self.label_6.setGeometry(QtCore.QRect(200, 140, 321, 41))
        font = QtGui.QFont()
        font.setFamily("Raleway SemiBold")
        font.setPointSize(15)
        font.setBold(True)
        font.setWeight(75)
        self.label_6.setFont(font)
        self.label_6.setAlignment(QtCore.Qt.AlignLeading|QtCore.Qt.AlignLeft|QtCore.Qt.AlignVCenter)
        self.label_6.setObjectName("label_6")
        self.label_7 = QtWidgets.QLabel(self.frame_3)
        self.label_7.setGeometry(QtCore.QRect(170, 80, 351, 31))
        font = QtGui.QFont()
        font.setFamily("Raleway SemiBold")
        font.setPointSize(15)
        font.setBold(True)
        font.setWeight(75)
        self.label_7.setFont(font)
        self.label_7.setObjectName("label_7")
        self.label_8 = QtWidgets.QLabel(self.frame_3)
        self.label_8.setGeometry(QtCore.QRect(210, 30, 311, 31))
        font = QtGui.QFont()
        font.setFamily("Raleway SemiBold")
        font.setPointSize(15)
        font.setBold(True)
        font.setWeight(75)
        self.label_8.setFont(font)
        self.label_8.setObjectName("label_8")
        profile.setCentralWidget(self.centralwidget)
        self.statusbar = QtWidgets.QStatusBar(profile)
        self.statusbar.setObjectName("statusbar")
        profile.setStatusBar(self.statusbar)

        self.retranslateUi(profile)
        self.close.clicked.connect(profile.close)
        self.min.clicked.connect(profile.showMinimized)
        self.max.clicked.connect(profile.showNormal)
        QtCore.QMetaObject.connectSlotsByName(profile)

    def retranslateUi(self, profile):
        _translate = QtCore.QCoreApplication.translate
        profile.setWindowTitle(_translate("profile", "MainWindow"))
        self.lineEdit.setPlaceholderText(_translate("profile", "type new Username"))
        self.lineEdit_2.setPlaceholderText(_translate("profile", "type new E-mail"))
        self.lineEdit_3.setPlaceholderText(_translate("profile", "type new Password"))
        self.pushButton.setText(_translate("profile", "Edit"))
        self.close.setToolTip(_translate("profile", "close"))
        self.label.setText(_translate("profile", "Profile"))
        self.max.setToolTip(_translate("profile", "normal"))
        self.min.setToolTip(_translate("profile", "minimize"))
        self.pushButton_2.setText(_translate("profile", "Home"))
        self.pushButton_2.setShortcut(_translate("profile", "Home"))
        self.pushButton_3.setText(_translate("profile", "Downloader"))
        self.pushButton_4.setText(_translate("profile", "About Me"))
        self.label_3.setText(_translate("profile", "current Username:"))
        self.label_4.setText(_translate("profile", "current E-mail:"))
        self.label_5.setText(_translate("profile", "current Password:"))
        self.label_6.setText(_translate("profile", "Null"))
        self.label_7.setText(_translate("profile", "Null"))
        self.label_8.setText(_translate("profile", "Null"))
